### DISCUSSION TITLE
combined_cycle_chp

### DISCUSSION THREAD

**Discussion by doimattertome** [oemof/tespy#512] [Q&A]
Hello! I'm here again, I've been trying to follow a tutorial on modeling a gas steam cycle cogeneration unit, but I've been having a lot of problems with the code I've written, and the most recent error I got was: You cannot specify two or more values for mass flow in the same linear branch ( You cannot specify two or more values for mass flow in the same linear branch ( starting at Merge3 and ending at Merge1). But I have checked that the connected flows between the two are equal everywhere.
Also I don't know how many parameters I need to calculate it correctly, can you point me to them, I've attached my code file and a diagram of the cogeneration unit.thank you very much for your help, good luck!
[Uploading chp.txt…]()
![Uploading 图片1.png…]()

**Comment by fwitte** [comment]
Hi @doimattertome,

thank you for reaching out, both your code (please add it by directly adding the code as text enquted by ` ```python [...]``` `) and the flow sheet have not been attached properly it seems... Can you update those?

Best
